packages/core/src/App/Components/Layout/Footer/responsible-trading.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Icon, Popover } from '@deriv/components';
+import { getDerivComLink } from '@deriv/shared/utils/url';
+import { localize } from '@deriv/translations';
+
+export const ResponsibleTrading = () => {
+    return (
+        <div className='footer__link'>
+            <a href={getDerivComLink('/')} target='_blank' rel='nofollow noreferrer'>
```

Comment:
should it redirect to the responsible trading page? `deriv.com/responsible-trading`

Reply:
Oh, I thought we do not have that page ready. Will do it. thanks for pointing it out.